modin/core/io/column_stores/column_store_dispatcher.py

```diff
@@ -128,9 +128,7 @@ def build_index(cls, partition_ids):
         else:
             index = index_len
             index_len = len(index)
-        index_chunksize = compute_chunksize(
-            pandas.DataFrame(index=index), num_partitions, axis=0
-        )
+        index_chunksize = compute_chunksize((len(index), None), num_partitions, axis=0)
```

Comment:
This feels a little hacky, instead of passing a tuple should we pass the values separately?

Reply:
I guess this is done to be able to pass a simple `df.shape` without unpacking it in the caller.

Reply:
@vnlitvinov exactly

Reply:
I don't think it would be very different to have them as separate variables that can default to `None`. It would be `*df.shape` to unpack.

Reply:
Suppose these arguments are as follows: `def compute_chunksize(row_count=None, column_count=None, num_splits, ...)`. `num_splits` is a non-optional variable that need to be placed before optional arguments. What do you think?

Reply:
I am okay with this interface. Also, we could make `num_splits` optional too and if it is not specified, `NPartitions.get()` is used by default.

Side note: I wonder should we have `axis` parameter in that case (`row_count`/`col_count` could be enough)?

Reply:
@devin-petersohn @YarShev the signature is updated.

Previous version is better IMHO.